src/aiida/calculations/monitors/base.py

```diff
@@ -4,12 +4,13 @@
 
 import tempfile
 from pathlib import Path
+from typing import Union
 
 from aiida.orm import CalcJobNode
-from aiida.transports import Transport
+from aiida.transports import AsyncTransport, Transport
 
 
-def always_kill(node: CalcJobNode, transport: Transport) -> str | None:
+def always_kill(node: CalcJobNode, transport: Union['Transport', 'AsyncTransport']) -> str | None:
```

Comment:
```suggestion
def always_kill(node: CalcJobNode, transport: Transport | AsyncTransport) -> str | None:
```

Reply:
Same for other places, but it is fine to leave it as this, I think we will find a time to change them all. So please ignore my comment above.